lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -1244,40 +1219,18 @@ struct StoreOpConversion
                              rewriter.create<LLVM::ConstantOp>(
                                  loc, u32Ty, IntegerAttr::get(u32Ty, elemIdx)));
         }
-        llWord = bitcast(llWord, valArgTy);
+        llWord = bitcast(llWord,
```

Comment:
I think it's better to let the whole statement within a macro block.

Reply:
Fixed.